rholang/src/main/scala/coop/rchain/rholang/interpreter/PrettyPrinter.scala

```diff
@@ -166,9 +158,9 @@ case class PrettyPrinter(freeShift: Int, boundShift: Int) {
     }
 
   private def buildVariables(bindCount: Int): String =
-    buildSeq(
-      (0 until Math.min(MAX_NEW_VAR_COUNT, bindCount))
-        .map(i => GPrivate(s"x${boundShift + i}")))
+    (0 until Math.min(MAX_NEW_VAR_COUNT, bindCount))
+      .map(i => s"x${boundShift + i}")
+      .mkString(", ")
```

Comment:
I wonder if the mkString pattern wouldn't clean up other parts of this code.

Again, not a blocker.